annotations/decorators.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib.auth.decorators import login_required
+from django.core.exceptions import PermissionDenied
+from functools import wraps
+
+def vogon_admin_required(view_func):
```

Comment:
is there a situation where a Django admin (staff) would be denied access but a Vogon admin granted access? Based on the Admin User management access toggling, it seems that Django admin trumps Vogon admin...